src/components/CharacterPreview.js

```diff
@@ -194,7 +194,7 @@ export function CharacterPreview(props) {
             </Flex>
 
             <Flex vertical>
-              <StatText style={{ fontSize: 18, fontWeight: 400, textAlign: 'center' }} ellipsis={true}>
```

Comment:
I'm adding the ellipsis={true} back in as a personal preference, it kinda looks bad when a whole line is taken up by a single word bc the light cone name is too long